pyreleaseplugin/release.py

```diff
@@ -202,19 +205,21 @@ class ReleaseCommand(Command):
 
     user_options = [
         ("version=", "v", "new version number"),
-        ("description=", "d", "a description of the work done in the release"),
         ("version-file=", "f", "a Python file containing the module version number"),
-        ("changelog-file=", "f", "a Markdown file containing a log changes"),
```

Comment:
So previously the `f` option was for both `version-file` and `changelog-file`? How did that ever work?

Reply:
Never worked! But I was never using the abbreviated option name anywhere, so I never noticed.